docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  typesense:
+    stdin_open: true
+    tty: true
+    container_name: typesense
+    environment:
+      - TYPESENSE_DATA_DIR=/data
+      - TYPESENSE_API_KEY=xyz
+    volumes:
+      - ./data:/data
```

Comment:
No need to manually create the data folder, as in `mkdir /tmp/typesense-server-data`.